cmake/dependencies/CMakeLists.txt

```diff
@@ -104,6 +104,48 @@ if(BUILD_GLM)
   message(CHECK_PASS "fetched")
 endif()
 
+# ##############################################################################
+# nlohmann/json
+# ##############################################################################
+message(CHECK_START "Fetching nlohmann/json")
+list(APPEND CMAKE_MESSAGE_INDENT "  ")
+
+FetchContent_Declare(
+  nlohmann_json 
+  GIT_REPOSITORY "https://github.com/nlohmann/json.git"
+  GIT_TAG "v3.11.3"
+  UPDATE_DISCONNECTED 1)
+FetchContent_MakeAvailable(nlohmann_json)
+
+list(POP_BACK CMAKE_MESSAGE_INDENT)
+message(CHECK_PASS "fetched")
+
+# ##############################################################################
+# assimp 
+# ##############################################################################
+if(BUILD_ASSIMP)
+  message(CHECK_START "Fetching assimp")
+  list(APPEND CMAKE_MESSAGE_INDENT "  ")
+
+  set(ASSIMP_BUILD_TESTS
+      OFF
+      CACHE BOOL "Do not build assimp tests" FORCE)
+
+  set(ASSIMP_INSTALL
+      OFF
+      CACHE BOOL "Do not install assimp" FORCE)
+      
+  FetchContent_Declare(
+  assimp 
+  GIT_REPOSITORY "https://github.com/assimp/assimp.git"
+  GIT_TAG "v5.4.3"
+  UPDATE_DISCONNECTED 1)
+  FetchContent_MakeAvailable(assimp)
+
+  list(POP_BACK CMAKE_MESSAGE_INDENT)
+  message(CHECK_PASS "fetched")
+endif()
```

Comment:
```
[cmake] --   Enabled importer formats: AMF 3DS AC ASE ASSBIN B3D BVH COLLADA DXF CSM HMP IRRMESH IQM IRR LWO LWS M3D MD2 MD3 MD5 MDC MDL NFF NDO OFF OBJ OGRE OPENGEX PLY MS3D COB BLEND IFC XGL FBX Q3D Q3BSP RAW SIB SMD STL TERRAGEN 3D X X3D GLTF 3MF MMD
[cmake] --   Disabled importer formats: USD
[cmake] --   Enabled exporter formats: OBJ OPENGEX PLY 3DS ASSBIN ASSXML M3D COLLADA FBX STL X X3D GLTF 3MF PBRT ASSJSON STEP
[cmake] --   Disabled exporter formats:
```
First of all, I don't think we need _any_ importers. Second of all we probably don't plan to support exporting to all of those formats, we should only enable importers which we need